hoodie-client/src/test/java/com/uber/hoodie/common/HoodieTestDataGenerator.java

```diff
@@ -115,6 +115,17 @@ public static GenericRecord generateGenericRecord(String rowKey, String riderNam
     return rec;
   }
 
+  /**
+   * Generates a new avro record of the above schema format, with the given hoodiekey, riderName and driverName
+   */
+  public static TestRawTripPayload generateGenericRecord(HoodieKey key, String commitTime,
```

Comment:
This is returning a `TestRawTripPayload` .. Can we reuse generateRandomValue() above.. any specific reason you need rider and driver names specifically passed in and not use the defaults there?